modules/serverless-installing-the-serverless-operator.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+= Installing the OpenShift Serverless Operator
+
+The OpenShift Serverless Operator can be installed by using the operator indexed in the OpenShift OperatorHub. 
```

Comment:
"The OpenShift Serverless Operator can be installed by using the {product-title} instructions on installing an Operator."

Include CLI and web console modules?